.github/workflows/update-docs.yml

```diff
@@ -50,3 +66,9 @@ jobs:
           git add -A
           git commit --allow-empty -m "Auto update docs" -a
           git push
+
+      - name: Delete Python Stubs
+        uses: geekyeggo/delete-artifact@v5
+        with:
+          name: PythonStubs
+          failOnError: false
```

Comment:
better move to common job of removing all artifacts

Reply:
fixed